src/site/_includes/components/description_list/description_list.scss

```diff
@@ -0,0 +1,54 @@
+.description_list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: center;
+  max-width: 100%;
+}
+
+.description_item {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  width: 100%;
+  margin-right: $spacing-medium;
+
```

Comment:
Add: align-content: flex-start. Makes the text align to the top of the element.

Reply:
Done!